common/display/kmsfencehandler.cpp

```diff
@@ -82,6 +90,11 @@ void KMSFenceEventHandler::HandleRoutine() {
     close(kms_fence_);
     kms_fence_ = -1;
   }
+
+  for (NativeSurface *surface : surfaces_) {
+     surface->SetInUse(false);
```

Comment:
Please clarify how its guarantee that the previous plane state layers are not displayed ? anyway we do async flip right after commit the actual mmio update of the display registers will happen at the next vblank only. Even for the nativesurface same applicable right how will know the previous native surface is no more scanout buffer